spec/integration/multi_xml/xml_spec.rb

```diff
@@ -1,7 +1,13 @@
 require 'spec_helper'
 
 describe Grape::Xml do
```

Comment:
This shouldn't be necessary, integration specs run in isolation. Is this failing on HEAD?